### PR TITLE
(maint) modsync update - master

### PR DESCRIPTION
**Pull request by glennsarti**
The PR contains the updates from the new modsync config for windows;

- LICENSE, NOTICE, CONTRIBUTING, MAINTAINERS updates (where applicable)
- Gemfile, Appveyor, Travis, gitignore cleanup

https://github.com/puppetlabs/modulesync_configs/pull/66
https://github.com/puppetlabs/modulesync_configs/pull/63
https://github.com/puppetlabs/modulesync_configs/pull/62
https://github.com/puppetlabs/modulesync_configs/pull/61

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Puppet Module - puppetlabs-dsc
+
+Copyright 2015 - 2016 Puppet, Inc.
```
Comment (ferventcoder) [inline NOTICE]: You knew I was going to tank this one as well ;)

Comment (ferventcoder) [inline NOTICE]: https://github.com/puppetlabs/puppetlabs-dsc#license

Shorthand - 

~~~md
Copyright 2014 - 2016 Marc Sutter, original author
Copyright 2015 - 2016 Puppet, Inc.
~~~

Comment (glennsarti) [inline NOTICE]: Updated

File: .sync.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@ Gemfile:
       - gem: cim
       - gem: mof
         git: 'https://github.com/puppetlabs/mof.git'
+        ref: 'f50581901c53ff6a40c54b72ef5f4fcaed9679a1'
```
Comment (ferventcoder) [inline .sync.yml]: wheeeeeee!

**Pull request merged**